dev/tools/update_icons.dart

```diff
@@ -450,6 +492,8 @@ class Icon {
   String get dartDoc =>
       '<i class="material-icons$htmlSuffix md-36">$shortId</i> &#x2014; $family icon named "$name"$style';
```

Comment:
nit: not sure if there should be space between name and style

Reply:
the space is already included, all good